app/models/manageiq/providers/redhat/inventory/persister/infra_manager.rb

```diff
@@ -29,6 +29,7 @@ def initialize_inventory_collections
     add_collection(infra, :root_folder_relationship)
     add_collection(infra, :vm_resource_pools)
     add_collection(infra, :vm_parent_blue_folders)
+    add_collection(infra, :distributed_virtual_switches, model_class: ManageIQ::Providers::InfraManager::DistributedVirtualSwitch)
```

Comment:
If we add a subclass of DistributedVirtualSwitch under Ovirt::InfraManager you won't need this model_class either